numba/np/math/mathimpl.py

```diff
@@ -0,0 +1,453 @@
+"""
+Provide math calls that uses intrinsics or libc math functions.
+"""
+
+import math
+import operator
+import sys
+import numpy as np
+
+import llvmlite.ir
+from llvmlite.ir import Constant
+
+from numba.core.imputils import Registry, impl_ret_untracked
+from numba import typeof
+from numba.core import types, utils, config, cgutils
+from numba.core.extending import overload
+from numba.core.typing import signature
+from numba.cpython.unsafe.numbers import trailing_zeros
+
+
+registry = Registry('mathimpl')
+lower = registry.lower
```

Comment:
Same comment about `lower`. Should all `@lower` be commented out for now?

Reply:
Fixed in e91e8ca